src/ripple/protocol/impl/BuildInfo.cpp

```diff
@@ -76,6 +77,77 @@ getFullVersionString()
     return value;
 }
 
+std::uint64_t
+getEncodedVersion()
+{
+    static std::uint64_t const cookie = []() {
+        std::uint64_t c = 0x183B000000000000;
```

Comment:
May be 'ull' prefix?

Reply:
@HowardHinnant, @scottschurr, any suggestions here?

Reply:
I like the `ull` suffix here.  You could try it as a prefix, but I don't think it will compile. :-)  That being said, it's optional.  Just a readability issue.